src/core/lib/address_utils/sockaddr_utils.cc

```diff
@@ -236,6 +244,7 @@ std::string grpc_sockaddr_to_uri(const grpc_resolved_address* resolved_addr) {
   if (scheme == nullptr || strcmp("unix", scheme) == 0) {
     return grpc_sockaddr_to_uri_unix_if_possible(resolved_addr);
   }
+  // TODO(anramach): Encode the string using URI::Parse before returning.
```

Comment:
I think you mean `URI::Create()` and `URI::ToString()`, not `URI::Parse()`.

Reply:
Yes, thanks for catching that. Fixed!